public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -139,7 +145,11 @@ function FolderGroupAndEvaluationInterval({
               <Stack direction="column" gap={1}>
                 {getValues('group') && getValues('evaluateEvery') && (
                   <span>
-                    All rules in the selected group are evaluated every {evaluateEvery}.{' '}
+                    <Trans i18nKey="alert-rule-form.evaluation-behaviour-group.text">
+                      All rules in the selected group are evaluated every
+                    </Trans>
+                    {evaluateEvery}
+                    <Trans i18nKey="alert-rule-form.evaluation-behaviour-group.point">.</Trans>{' '}
```

Comment:
```suggestion
                    <Trans i18nKey="alert-rule-form.evaluation-behaviour-group.text" evaluateEvery={evaluateEvery}>
                      All rules in the selected group are evaluated every {{evaluateEvery}}.
                    </Trans>
```
I think you'd be able to pass variables to translation component instead of creating a separate one just for the dot.

Reply:
ah!! thanks @Clarity-89